config/qt5_qml.mpb

```diff
@@ -0,0 +1,5 @@
+project: qt5_gui {
+  libs += Qt5Qml
+  includes += $(QT5_INCDIR)/QtQml
```

Comment:
Are the in includes necessary, none of the user qt5 projects do that, maube add QtQml to the include in code?

Reply:
yes it‘s necessary ，and the other mpb should add this，because the recommended usage in Qt example is include the header directly not with the module name， for example， when use QLineEdit, the recommended import fashion is #include <QLineEdit> not #include<QtWidgets/QLineEdit>,  at least from Qt5.6 and both under linux and windows this is the truth.

Reply:
I did find examples on the Qt website that do use the module as prefix, does seem much safer and clearer.